docs/examples/3rdparty_analysis_workflow.sh

```diff
@@ -89,6 +89,12 @@ datalad update alice --merge
 datalad install result.txt
```

Comment:
No, the line you referenced obtains the change made by alice in bob's dataset, hence all is ready for publish and apart from data availability there is no change.